ERROR: Sending notification: Successfully update to AlmaLinux 8

noticed at the very end of an update

```
[ERROR] Sending notification: Successfully update to AlmaLinux 8
```

```
> /scripts/elevate-cpanel --status
success
```

```
* 07-22:42:43 (3648) [INFO] Complete!
* 07-22:42:43 (3658) [INFO]
* 07-22:42:43 (1268) [INFO] ******************************************************************************************
* 07-22:42:43 (1269) [INFO] *
* 07-22:42:43 (1270) [INFO] * Great SUCCESS! Your upgrade to AlmaLinux 8 is complete.
* 07-22:42:43 (1271) [INFO] *
* 07-22:42:43 (1272) [INFO] ******************************************************************************************
* 07-22:42:43 (585) [ERROR] Sending notification: Successfully update to AlmaLinux 8
* 07-22:42:43 (586) [ERROR] The cPanel & WHM server has completed the elevation process from CentOS 7 to AlmaLinux 8.

The update to AlmaLinux 8 was successful but please note that one ore more notifications require your attention:

* The following kernels should probably be removed as they will not function on AlmaLinux 8:

    kernel-3.10.0-1160.71.1.el7.x86_64
    kernel-3.10.0-1160.76.1.el7.x86_64

You can remove these by running: /usr/bin/rpm -e kernel-3.10.0-1160.71.1.el7.x86_64 kernel-3.10.0-1160.76.1.el7.x86_64


* The following recommendations should now be reviewed for further action:
The following kernels should probably be removed as they will not function on AlmaLinux 8:

    kernel-3.10.0-1160.71.1.el7.x86_64
    kernel-3.10.0-1160.76.1.el7.x86_64

You can remove these by running: /usr/bin/rpm -e kernel-3.10.0-1160.71.1.el7.x86_64 kernel-3.10.0-1160.76.1.el7.x86_64



* 07-22:42:43 (1268) [INFO] ******************************************************************************************
* 07-22:42:43 (1269) [INFO] *
* 07-22:42:43 (1270) [INFO] * Doing final reboot
* 07-22:42:43 (1271) [INFO] *
* 07-22:42:43 (1272) [INFO] ******************************************************************************************
* 07-22:42:43 (3637) [INFO] Running: /usr/sbin/reboot now
* 07-22:42:43 (3638) [INFO]
```

--- a/elevate-cpanel
+++ b/elevate-cpanel
@@ -597,7 +597,7 @@ EOS
         }
     }
 
-    send_notification( qq[Successfully update to $pretty_distro_name] => $msg );
+    send_notification( qq[Successfully updated to $pretty_distro_name] => $msg, is_success => 1 );
 
     return;
 }
@@ -629,10 +629,10 @@ EOS
     return;
 }
 
-sub send_notification ( $subject, $msg ) {
+sub send_notification ( $subject, $msg, %opts ) {
 
     eval {
-        _send_notification( $subject, $msg );
+        _send_notification( $subject, $msg, %opts );
         1;
     }
       or warn "Fail to send notification: $@";
@@ -640,13 +640,20 @@ sub send_notification ( $subject, $msg ) {
     return;
 }
 
-sub _send_notification ( $subject, $msg ) {
+sub _send_notification ( $subject, $msg, %opts ) {
+
+    my $is_success = delete $opts{is_success};
 
     # note: no need to use one iContact::Class this is a one shot message
     require Cpanel::iContact;
 
-    ERROR("Sending notification: $subject");
-    ERROR($msg);
+    INFO("Sending notification: $subject");
+
+    my $log   = $is_success ? \&INFO : \&ERROR;
+    my @lines = split( "\n", $msg );
+    foreach my $line (@lines) {
+        $log->($line);
+    }
 
     Cpanel::iContact::icontact(
         'application' => 'elevate',

--- a/t/notify.t
+++ b/t/notify.t
@@ -23,10 +23,11 @@ use cPstrict;
 my $notification;
 my $mock_cpev = Test::MockModule->new('cpev');
 $mock_cpev->redefine(
-    send_notification => sub ( $title, $body ) {
+    send_notification => sub ( $title, $body, %opts ) {
         $notification = {
             title => $title,
             body  => $body,
+            opts  => \%opts,
         };
         return;
     }
@@ -38,17 +39,22 @@ my $cpev = bless {}, 'cpev';
 
 $cpev->_notify_success();
 
-is $notification->{title}, 'Successfully update to AlmaLinux 8',                                                            '_notify_success: title';
-is $notification->{body},  qq[The cPanel & WHM server has completed the elevation process from CentOS 7 to AlmaLinux 8.\n], '_notify_success: body';
+is $notification, {
+    title => 'Successfully updated to AlmaLinux 8',
+    body  => qq[The cPanel & WHM server has completed the elevation process from CentOS 7 to AlmaLinux 8.\n],
+    opts  => { is_success => 1 },
+  },
+  " _notify_success"
+  or diag explain $notification;
 
 $notification = {};
 
-ok cpev::add_final_notification("My First Notification");
+ok cpev::add_final_notification("My First Notification"), 'add_final_notification';
 
 is cpev::read_stage_file(), { final_notifications => ['My First Notification'] }, "stage file - notifications";
 
-ok !cpev::add_final_notification(undef);
-ok !cpev::add_final_notification('');
+ok !cpev::add_final_notification(undef), q[cannot add_final_notification(undef)];
+ok !cpev::add_final_notification(''),    q[cannot add_final_notification('')];
 
 is cpev::read_stage_file(), { final_notifications => ['My First Notification'] }, "stage file - notifications";
 
@@ -65,8 +71,8 @@ is cpev::read_stage_file(), {
 
 $cpev->_notify_success();
 
-is $notification->{title}, 'Successfully update to AlmaLinux 8', '_notify_success: title';
-is $notification->{body},  <<EOS,                                '_notify_success: body' or note $notification->{body};
+is $notification->{title}, 'Successfully updated to AlmaLinux 8', '_notify_success: title';
+is $notification->{body},  <<EOS,                                 '_notify_success: body' or note $notification->{body};
 The cPanel & WHM server has completed the elevation process from CentOS 7 to AlmaLinux 8.
 
 The update to AlmaLinux 8 was successful but please note that one ore more notifications require your attention:
@@ -77,4 +83,5 @@ The update to AlmaLinux 8 was successful but please note that one ore more notif
 with two lines.
 EOS
 no_messages_seen();
+
 done_testing();

